priam/src/main/java/com/netflix/priam/notification/BackupNotificationMgr.java

```diff
@@ -102,6 +107,42 @@ private Map<String, MessageAttributeValue> getMessageAttributes(
                 new MessageAttributeValue()
                         .withDataType("String")
                         .withStringValue(backupFileType.name()));
+
+        String additionalAttrs =
+                this.backupRestoreConfig.getBackupNotificationAdditionalMessageAttrs();
+        if (!StringUtils.isBlank(additionalAttrs)) {
+            try {
+                List<String> attrs =
```

Comment:
Move this Splitting logic to BackupRestoreConfig. Be mindful to handle nulls properly.